tools/logger/convert.c

```diff
@@ -387,9 +388,30 @@ static void print_entry_params(const struct log_entry_header *dma_log,
 	if (raw_output)
 		use_colors = 0;
 
+	/* Something somewhere went wrong */
 	if (dt < 0 || dt > 1000.0 * 1000.0 * 1000.0)
 		dt = NAN;
 
+	static int entry_number = 1;
+	static uint64_t timestamp_origin;
+
+	/* The first entry:
+	 *  - is never shown with a relative TIMESTAMP (to itself!?)
```

Comment:
It looks like mixed declaration and code

Reply:
@marc-hb cxan you follow coding standard until standard change is agreed by TSC.  Thanks.

Reply:
Done. I'm also adding `-Wdeclaration-after-statement` in PR #3995 so history stops repeating itself. Let's stop wasting most precious code review time with stuff computers can do.